lib/entry.ts

```diff
@@ -140,13 +140,26 @@ type IDBPDatabaseExtends = Omit<
   'createObjectStore' | 'deleteObjectStore' | 'transaction' | 'objectStoreNames'
 >;
 
+/**
+ * A variation of DOMStringList with precise string types
+ */
+export interface TypedDOMStringList<
+  T extends string
+> extends DOMStringList {
+    readonly length: number;
+    contains(string: T): boolean;
```

Comment:
Do you think the type should be restricted to `T` here? Passing a different string doesn't feel like it should result in a type error.

Reply:
`.includes` for an array is also restricted to `T` so this stays consistent with it.